sql-statements/sql-statement-select.md

```diff
@@ -95,6 +91,13 @@ TableList ::=
 
 ![WindowClauseOptional](/media/sqlgram/WindowClauseOptional.png)
 
+
+**TableSampleOpt**
+
+```ebnf+diagram
```

Comment:
It seems that github can't render ebnf+diagram as a picture
https://github.com/pingcap/docs/blob/98d937fc375d9be18d57281e9caa2678ef23541d/sql-statements/sql-statement-select.md

Reply:
https://kennytm.github.io/website-docs/dist/

![image](https://user-images.githubusercontent.com/1272980/227141210-73ca4754-e5ef-4f20-8ccf-9024e547696c.png)

Reply:
@kennytm is this something we can add to `CONTRIBUTING.md` ?

Reply:
See also #4389

Reply:
> https://kennytm.github.io/website-docs/dist/

This tool is amazing. 👍  I'll make sure we add this to `CONTRIBUTING.md`!